src/app/dashboard/Missions/MissionEditView.jsx

```diff
@@ -211,9 +217,17 @@ const MissionEditView = ({ mission, toDetailsView, toListView }) => {
     date: values.deliveryWindow.startTime,
     location: "",
   });
+  const [selectedVolunteer, setSelectedVolunteer] = useState(
+    volunteers.find((el) => el.id === mission.volunteerUid)
+  );
 
   const props = { classes, mission };
 
+  const volunteerEditable =
+    mission.status === MissionStatus.unassigned ||
+    mission.status === MissionStatus.tentative ||
+    mission.status === MissionStatus.assigned;
+
```

Comment:
outside of the class
`const volunteerStatus = [MissionStatus.unassigned, MissionStatus.tentative, MissionStatus.assigned]`
inside of the class
`const volunteerEditable = volunteerStatus.includes(mission.status)`

Reply:
Fixed!